src/test/java/com/origamisoftware/teach/agile/junit/SubscriptionPeriodTest.java

```diff
@@ -5,8 +18,7 @@
 
 import java.util.Calendar;
 
-import static junit.framework.Assert.assertEquals;
-import static junit.framework.Assert.fail;
+import static junit.framework.Assert.*;
 
```

Comment:
do not use * instead use explicit class names. Why? 1) makes code easier to read - reader knows precisely what classes are being used  here. 2) makes comparing (diffing) versions of code easier.